### ISSUE TITLE
Typo on page death-rates

### ISSUE THREAD

**Issue by Totktonada** [whatifrussian/website#201]
Typo in http://chtoes.li/page/death-rates

> По крайней мере должен, если рассчитать математически. **Интуитивно кажется очевидным, что уровень смертности ниже 10 из 1 000 недостижим.** Если каждый год умирает менее, чем один человек на сотню

Reader’s comment: Здесь наверно имеется ввиду "уровень смертности выше"

**Comment by Totktonada** [comment]
Нет, здесь именно «ниже». Но слово «недостижим» плохо здесь подходит (в оригинале было unsustainable). Точным переводом здесь было бы «нежизнеспособен», но к уровню смертности это понятие кажется слабо применимым… хотя звучит весьма забавно. Может, стоит так и написать.

И еще «из» в «10 из 1000», возможно, стоит заменить на «на».

**Comment by librarian** [comment]
Reviewed decision: close. Closing as completed.
